docker-compose.yml

```diff
@@ -23,6 +23,7 @@ services:
       - AB2D_DB_HOST=db
       - AB2D_DB_PORT=5432
       - AB2D_DB_DATABASE=ab2d
+      - AB2D_DB_SSL_MODE=allow
```

Comment:
Does verify-full make more sense since it will do more verification for us or will that mess things up when we don't have an actual vert like in dev? https://jdbc.postgresql.org/documentation/head/ssl-client.html Maybe it's for for when using a prod env.

Reply:
My understanding after talking to @dkrylovsb  is that the `sslmode` would be turned on only in specific environments (such as for eg, prod).